opennms-base-assembly/src/main/filtered/etc/otrs.properties

```diff
@@ -52,6 +52,9 @@
 # Typical OTRS v 3.1.x and above endpoint
 #otrs.endpoint=http://localhost/otrs/nph-genericinterface.pl/Webservice/GenericTicketConnector
 
+# OTRS v 3.1.x and above require a soapAction HTTP header
+otrs.soapaction=http://www.otrs.org/TicketConnector/
```

Comment:
I don't think so. At least, it doesn't hurt to leave it uncommented.

I guess the `otrs.endpoint` property is commented because there are 2 alternatives and one _must_ choose one otherwise integration won't work.

On the other hand, `otrs.soapaction` will only be used when the 3.1+ integration is used, and will be ignored if the 3.1- integration is used.